codalab/lib/codalab_manager.py

```
@@ -57,7 +57,11 @@
 from codalab.model.worker_model import WorkerModel
 
 
-MAIN_BUNDLE_SERVICE = 'https://worksheets.codalab.org'
+try:
+    MAIN_BUNDLE_SERVICE = os.environ['MAIN_BUNDLE_SERVICE']
+except:
+    print('Please export MAIN_BUNDLE_SERVICE=http://bundle_service_host\n')
+    MAIN_BUNDLE_SERVICE = None
 
```

swallowing exceptions like this is bad! 
You can have swallow (ie. catch and ignore) exception only if exception is not problematic.
Clearly, the exception here is problematic. Code cant run if bundle service url is not set.

Suggestion:
```python
MAIN_BUNDLE_SERVICE  = os.environ.get('MAIN_BUNDLE_SERVICE')
if not MAIN_BUNDLE_SERVICE:
  raise Exception('Please export MAIN_BUNDLE_SERVICE=http://bundle_service_host')
```

---

The reason why I use try except is, if you do not export MAIN_BUNDLE_SERVICE before you run the code, os.environ.get('MAIN_BUNDLE_SERVICE') will raise an error. Then if not MAIN_BUNDLE_SERVICE: will be meaningless.

---

Right! 
then the `except:` will catch it, right?
does it halt the program? No
Should it halt the program? Yes!

---

os.environ is a dictionary.  See the difference between

os.environ['key'] # this one raises KeyError
os.environ.get('key')  # this one returns None